sklearn/ensemble/_forest.py

```diff
@@ -1177,6 +1177,8 @@ class RandomForestClassifier(ForestClassifier):
     A random forest is a meta estimator that fits a number of decision tree
     classifiers on various sub-samples of the dataset and uses averaging to
     improve the predictive accuracy and control over-fitting.
+    Trees in the forest use the best split strategy, i.e. equivalent to passing `splitter="best"`
```

Comment:
The linter is failing because this is more than 88 characters. Could your rewrap the sentence.

Reply:
Sorry about that, I fixed both paragraphs now.